### DISCUSSION TITLE
Selecpicker menu doesn't following select when component in overflow

### DISCUSSION THREAD

**Discussion by yayakyogi** [rsuite/rsuite#3532] [General]
how to resolve it?

**Comment by yayakyogi** [comment]
like this
![image](https://github.com/rsuite/rsuite/assets/57065482/0a2610fe-9beb-4a43-a403-fe1e8ddb4541)

**Comment by simonguo** [comment]
Hi @yayakyogi,  you can set a `container` prop for SelectPicker to render it in the specified container. It can follow the container to scroll.

```tsx
const App = () => {
  const ref = useRef();
  return (
    <div ref={ref} style={{ position: 'relative' }}>
      <SelectPicker container={() => ref.current} />
    </div>
  );
};
```
